### ISSUE TITLE
centered stauspanel

### ISSUE THREAD

**Issue by GrosBourrin** [MrOtherGuy/firefox-csshacks#282]
hello
FF 113
The thing i don't understand is when the page loads, the statuspanels appears correctly at center except the text that is always beginning at left to right. But when i hover a link the text is correctly centered ! what's wrong with me ?
here's my userchrome, the statuspanel block is at end : 
[userchrome [](url)](https://github.com/GrosBourrin/FIREFOX-BLUE-MOON/blob/master/userChrome.css)

corectly centered when i hover
 a link![cLEUkbZgDd](https://github.com/MrOtherGuy/firefox-csshacks/assets/40931468/c52470cf-335a-4957-8dbd-84b8d6e7181d)

left text display when page loading
![TpcEC9YQBs](https://github.com/MrOtherGuy/firefox-csshacks/assets/40931468/9e099746-42f0-4caa-90c8-efb7c16f1f1f)
```

#statuspanel:not([hidden]) {
	display: flex !important;
	justify-self: center !important;
	padding-bottom: 0 !important;
	inset-block: auto 0 !important;
	inset-inline: auto !important;
}

#statuspanel-label {
	background: #1E1D2D !important;
	font-family: "Fira Sans" !important;
	font-size: 13px !important;
	color: #ffff00 !important;
	border: 2px solid #1E1D2D !important;
	border-radius: 0px !important;
	margin-inline: auto !important;
}
```

**Comment by MrOtherGuy** [comment]
Apparently Firefox adds gives statuspanel some min-width while page is loading - this should be fixed with d9ed3a6

**Comment by GrosBourrin** [comment]
works nicely now ! thanks for the rewriting ! you're the best
